src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -62,29 +63,77 @@ class BaseSidebarScreen extends Component {
         Welcome.show({routes, showCreateMenu: this.showCreateMenu});
     }
 
+    componentDidUpdate(prevProps) {
+        if (!this.didScreenBecomeInactive(prevProps)) {
+            return;
+        }
+
+        // Hide menu manually when other pages are opened using shortcut key
+        this.hideCreateMenu();
+    }
+
+    /**
+     * Check if LHN became inactive from active status
+     *
+     * @param {Object} prevProps
+     * @return {boolean}
```

Comment:
Following the style guide

```suggestion
     * @return {Boolean}
```